sql/dd/dd_table.cc

```diff
@@ -2390,6 +2392,16 @@ std::unique_ptr<dd::Table> create_table(
   const dd::Object_table *dd_table =
       dict->get_dd_table(sch_obj.name(), table_name);
 
+  // During DDSE change, when create a myrocks table
+  // treat <upgrade>.<table> as dd table if mysql.<table> is a dd table
+  if (dd_table == nullptr && sch_obj.name() != MYSQL_SCHEMA_NAME.str &&
```

Comment:
Consider moving `sch_obj.name() != ...` just before `is_dd_table_name`, so that the cheaper checks go first